lxd/daemon_config.go

```diff
@@ -210,7 +210,8 @@ func daemonConfigInit(db *sql.DB) error {
 	for k, v := range dbValues {
 		_, ok := daemonConfig[k]
 		if !ok {
-			logger.Error("Found invalid configuration key in database", log.Ctx{"key": k})
+			logger.Warn("Found unknown configuration key in database", log.Ctx{"key": k})
```

Comment:
I wouldn't degrade it to Warn, having an unknown key in the DB shouldn't be possible at all unless you manually modified the DB or it's otherwise corrupted.